### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by Tetrerox**
Added tests for delete and deleteShift commands.

### CODE REVIEW

**Review by mweeruien — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/DeleteShiftCommand.java [hunk 1]
```diff
@@ -108,22 +109,15 @@ public CommandResult execute(Model model) throws CommandException {
...
-        // short circuit if same object
-        if (other == this) {
+    public boolean equals(Object o) {
+        if (this == o) {
             return true;
         }
-
-        // instanceof handles nulls
-        if (!(other instanceof DeleteShiftCommand)) {
+        if (o == null || getClass() != o.getClass()) {
```
Comment (Gabau) [inline src/main/java/seedu/address/logic/commands/DeleteShiftCommand.java:116]: I think it is better to use instanceof. I do not see a reason to use getClass?
```suggestion
        if (o == null || !(o instanceof DeleteShiftCommand)) {
```

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM. One question regarding the use of the getClass method. I think it is more consistent with the rest of the codebase to use instanceof instead of getClass.

**Review by Gabau — APPROVED** [review]
LGTM

**Pull request merged**